### DISCUSSION TITLE
Help understand options "Exclude .. words"

### DISCUSSION THREAD

**Discussion by w4hnsinn** [pymedusa/Medusa#10792] [Q&A] [answered]
I'm struggling to understand these options, found in Advanced Settings when editing specific shows:

> Exclude ignored words
> Exclude required words

Could somebody please explain what these options do? Either they do some very unexpected things, or they are written confusingly.

**Comment by p0psicles** [comment, marked as answer]
in `config - search settings` you'll find `ignored words` as well as `required words`. But you'll also find those on a per show level.
If you don't use the exclude option. Then it will add the per show words to the global words. If you toggle the exclude option, it will substract the per show words from the global words.

so global has words `a`, `b`, `c`
and per show has `a`.
the exclude option is enabled.
Then the result will be `b`, `c`

**Reply by w4hnsinn** [reply]
Thank you! I was right, that is not how I expected it to work :o but it makes sense now.
